cabal-install/src/Distribution/Client/ProjectConfig.hs

```diff
@@ -754,9 +754,9 @@ renderImplicitBadPackageLocation bpl = case bpl of
     BadLocGlobEmptyMatch pkglocstr ->
         "No cabal.project file or cabal file matching the default glob '"
```

Comment:
This one does not match well with `No <pkgname>.cabal was found and the current working directory is no...`

Reply:
Maybe an improvement would be print the default glob when no explicit cabal.project exists which afaik is `./*.cabal`. In fact `matching the default glob` talks about the cabal file and no the cabal.project file (it is a little bit ambiguous).

https://github.com/haskell/cabal/blob/b26e736a04da5fca30dfa50ea8cf16abf1c8314c/cabal-install/src/Distribution/Client/ProjectConfig.hs#L537

```suggestion
        "No cabal.project file in the current dir nor a cabal file matching the default glob (./*.cabal) '"
```

but making `./*.cabal` a constant used here and in the previous code using it.

Reply:
In fact this would make the error message more similar to the thrown by `cabal build`